cypress/e2e/unit/shortlist-overview.cy.js

```diff
@@ -105,35 +119,79 @@ context('signatures shortlist overview tests', () => {
+      documents: files2,
+    });
+
+    cy.createCase(caseTitle3);
+    cy.addSubcaseViaModal({
+      newCase: true,
+      agendaitemType: type3,
+      newShortTitle: subcaseTitleShort3,
+      longTitle: subcaseTitleLong3,
+      subcaseType: subcaseType3,
+      subcaseName: subcaseName3,
+      documents: files3,
     });
-    cy.addDocumentsToSubcase(files2);
 
     cy.createAgenda('Ministerraad', agendaDate);
     cy.openAgendaForDate(agendaDate);
+
     cy.addAgendaitemToAgenda(subcaseTitleShort1);
     cy.openDetailOfAgendaitem(subcaseTitleShort1);
     cy.changeDecisionResult('Goedgekeurd');
+
     cy.addAgendaitemToAgenda(subcaseTitleShort2);
     cy.openDetailOfAgendaitem(subcaseTitleShort2);
     cy.changeDecisionResult('Goedgekeurd');
-    cy.setAllItemsFormallyOk(3);
+
+    cy.addAgendaitemToAgenda(subcaseTitleShort3);
+    cy.openDetailOfAgendaitem(subcaseTitleShort3);
+    cy.changeDecisionResult('Goedgekeurd');
+
+    cy.setAllItemsFormallyOk(4);
     cy.approveDesignAgenda();
 
     cy.openDetailOfAgendaitem(subcaseTitleShort1);
     cy.addAgendaitemMandatee(mandatee1);
     cy.get(agenda.agendaitemNav.documentsTab).click();
     cy.get(document.documentCard.actions).click();
-    cy.get(document.documentCard.signMarking).click();
+    cy.intercept('POST', '/sign-flows').as('postSignFlows1');
+    cy.intercept('POST', '/sign-subcases').as('postSignSubcases1');
+    cy.get(document.documentCard.signMarking).click()
+      .wait('@postSignFlows1')
+      .wait('@postSignSubcases1');
+    cy.get(auk.loader).should('not.exist', {
+      timeout: 60000,
+    });
     cy.openDetailOfAgendaitem(subcaseTitleShort2);
     cy.addAgendaitemMandatee(mandatee2);
     cy.get(agenda.agendaitemNav.documentsTab).click();
     cy.get(document.documentCard.actions).click();
-    cy.get(document.documentCard.signMarking).click();
+    cy.intercept('POST', '/sign-flows').as('postSignFlows2');
+    cy.intercept('POST', '/sign-subcases').as('postSignSubcases2');
+    cy.get(document.documentCard.signMarking).click()
+      .wait('@postSignFlows2')
+      .wait('@postSignSubcases2');
+    cy.get(auk.loader).should('not.exist', {
+      timeout: 60000,
+    });
+    cy.openDetailOfAgendaitem(subcaseTitleShort3);
+    cy.addAgendaitemMandatee(mandatee3);
+    cy.get(agenda.agendaitemNav.documentsTab).click();
+    cy.get(document.documentCard.actions).click();
+    cy.intercept('POST', '/sign-flows').as('postSignFlows3');
+    cy.intercept('POST', '/sign-subcases').as('postSignSubcases3');
+    cy.get(document.documentCard.signMarking).click()
+      .wait('@postSignFlows3')
+      .wait('@postSignSubcases3');
+    cy.get(auk.loader).should('not.exist', {
+      timeout: 60000,
+    });
 
-    cy.setAllItemsFormallyOk(2);
+    cy.setAllItemsFormallyOk(3);
     cy.approveAndCloseDesignAgenda();
     cy.releaseDecisions();
-    cy.wait(20000); // shortlist not always found
+    cy.wait(60000); // shortlist not always found
```

Comment:
not sure if this fully needed, could keep for now and lower or remove when setup is part of the available testdata zip